src/reference/informative/informC02_units_reference.rst

```diff
@@ -8,4 +8,300 @@
 
   .. container:: infospec
 
-    **TODO**
\ No newline at end of file
+    .. container:: heading3
+
+      Understanding units references
+
+    Units references are different from other reference types in that their value could refer either to other units which you've imported or created in your model, or to one of the list of :ref:`built-in units<table_built_in_units>`.  
+
+    The trickier part is understanding them is during an import process, where the *scope* or *domain* in which named units exist becomes important, as alluded to in points :hardcodedref:`3.2.1 and 3.2.2.2`.
+
+    Consider the example below.
+    The first model :code:`BlueberryPieRecipe` simply combines two pre-made component ingredients; one for the crust and one for the filling.
+
+    .. code::
+
+      model: BlueberryPieRecipe
+        ├─ component: crust <╴╴╴╴╴╴╴╴╴╴╴╴╴╴╴╴╴╴╴╴╴┐
+        └─ component: filling <╴╴╴╴╴╴╴╴╴╴╴╴╴╴╴╴╴┐ ╷
+                                                ╷ ╷
+                                  imported components
+                                                ╵ ╵
+      # In filling_recipes.cellml:              ╵ ╵
+      model: PieFillingRecipes                  ╵ ╵
+        ├─ component: BlueberryCinnamonFilling ╶┘ ╵
+        └─ component: AppleAndPearFilling         ╵
+                                                  ╵
+      # In crust_recipes.cellml:                  ╵
+      model: PieCrustRecipes                      ╵
+        ├─ component: HazelnutLavenderCrust ╴╴╴╴╴╴┘
+        └─ component: CheeseAndAlmondCrust
+
+    .. container:: toggle
+
+      .. container:: header
+
+        See CellML syntax
+
+      .. code-block:: xml
+
+        <!-- Inside the file "how_to_make_blueberry_pie.cellml": -->
+        <model name="BlueberryPieRecipe">
+            <import xlink:href="path/to/my/crust_recipes.cellml">
+                <component name="crust" component_ref="HazelnutLavenderCrust" />
+            </import>
+            <import xlink:href="path/to/my/filling_recipe.cellml">
+              <component name="filling" component_ref="BlueberryCinnamonFilling" />
+            </import>
+        </model>
```

Comment:
Although it's quite clear to me from the path you use, you might want to use `relative/path/to/my/xxx.cellml` rather than `path/to/my/xxx.cellml`?

Reply:
41d3dbc